.gitignore

```diff
@@ -0,0 +1,77 @@
+# Temporary Build Files
```

Comment:
Please don't use one of those, it's a mess. IDE specific ignores should be in one's global gitignore. Would you mind dropping the generated part?